.github/workflows/pipeline.yml

```diff
@@ -42,3 +43,18 @@ jobs:
         fail_ci_if_error: true
         files: ./coverage.txt
         verbose: true
+  deploy:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to quay.io
+        uses: docker/build-push-action@v1
```

Comment:
It looks like there is a v2 of this action, should we start with that so we do not have to worry about upgrading later?
https://github.com/marketplace/actions/build-and-push-docker-images#upgrade-from-v1

Reply:
@vinny-sabatini fixed in commit 3fb3322565db73321246ef9c135b9cb42de00e71.